client/my-sites/plans-features-main/index.jsx

```diff
@@ -383,13 +383,9 @@ export class PlansFeaturesMain extends Component {
 
 	renderSingleProducts() {
 		const { displayJetpackPlans, translate } = this.props;
-		if ( ! displayJetpackPlans ) {
-			return null;
-		}
-
 		const displayBackup = isEnabled( 'plans/jetpack-backup' );
-		const displayScan = isEnabled( 'plans/jetpack-scan' );
-		if ( ! displayBackup && ! displayScan ) {
+
+		if ( ! displayJetpackPlans || ! displayBackup ) {
```

Comment:
A note here: while I'm fine with leaving this as-is, it would be nice to have it more flexible. Let's say we have 5 single product plans, and whichever is available, this condition should work. Not for this PR, just wanted to share my thoughts.